docs/usage/serialization.md

```diff
@@ -662,10 +665,11 @@ print(t.model_dump())
 #> {'id': '1234567890'}
 # TODO: this is wrong! not all of "user" should be excluded
 # TODO: do we need to fix the type of the argument to Field? Or do we want to just remove that functionality?
+print(t.model_dump(include={'id': True, 'value': True}))  # (1)!
+#> {'id': '1234567890'}
 ```
 
-Explicitly setting `exclude`/`include` on `model_dump` and `model_dump_json` takes priority over the
-`exclude`/`include` from the field constructor (i.e. `Field(..., exclude=True)`):
+1. `value` excluded from the output because it excluded in `Field`.
```

Comment:
is this supposed to be a numbered list?

Reply:
This will appear like

![pic1](https://github.com/pydantic/pydantic/assets/3122442/ae929a4a-8079-4e65-b5ac-dbd01acb7e5e)
![pic](https://github.com/pydantic/pydantic/assets/3122442/432ba0b6-14f1-4c5b-9534-5ff71acb018c)

Reply:
me being dumb :facepalm: